web/conf/rhn_web.conf

```diff
@@ -132,3 +132,5 @@ web.themes = susemanager-light,susemanager-dark,uyuni
 # Set the default webUI style theme name
 web.theme_default = uyuni
 
+# Disable the update status
+disable_update_status = 0
```

Comment:
Shouldn't this be `web.disable_update_status`?

Reply:
I tested it I think there's some mess... AFAIK this should be the behaviour: let's take parameter is `A.parameter`, it can be set:
- in `/etc/rhn/rhn.conf` as `A.parameter`
- in `/usr/share/rhn/config-defaults/rhn_web.conf` as `parameter`

basically I always supposed that  `A.parameter` in `rhn.conf` ==  `parameter in rhn_A.conf`

**The actual behaviour** that I'm seeing is different, like the prefix it's not evaluated at all. E.g. `web.theme_default = uyuni` can be set set as:

- `web.theme_default = uyuni` and `theme_default = uyuni` in `/etc/rhn/rhn.conf`
- `web.theme_default = uyuni` and `theme_default = uyuni` in `/usr/share/rhn/config-defaults/rhn_web.conf`

Since  I think the actual behaviour is not consistent, I prefer to use `disable_update_status`...but any idea is welcome :) (probably we should open a GH Issue for it)

Reply:
maybe @rjmateus can help with this

Reply:
Yes, it works regardless, but `rhn_java.conf` is consistent with the "java" prefix, and I did not noticed that that was not the case for `rhn_web.conf` (which I agree should be fixed to be consistent one way or another).

Why did you make it part of the `web` namespace, though? It means it will be packaged together with `spacewalk-web`, but since you're using this property in the java code only. I feel it would be a better fit for the `java` namespace.

Reply:
> Why did you make it part of the web namespace, though? It means it will be packaged together with spacewalk-web, but since you're using this property in the java code only. I feel it would be a better fit for the java namespace.


I always though `web` namespace was for UI settings and `java` for engine...but since there's param as `java.login_banner` and `java.custom_header`, I can move it to `java` namespace. Thanks for noticing it :)

Reply:
I think this is fine as `java.disable_list_update_status`